### PR TITLE
KinBP firmware

### PR DESCRIPTION
**Pull request by DmNosachev**
Firmware for [KinBP](https://github.com/DmNosachev/kinbp), a drop-in replacement controller for Kinesis Contoured keyboards.
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by DmNosachev** [comment]
Hello,
Lint is failing with following message: `Error: Process completed with exit code 255`.
But I don't understand the reason for the error. Locally, the Lint check completes successfully:

```
[nosac@WIN-OSD802N8KU1 qmk_firmware]$ qmk lint -kb kinesis/kinbp/blackpill411
Ψ Lint check passed!
[nosac@WIN-OSD802N8KU1 qmk_firmware]$ qmk lint -kb kinesis/kinbp/blackpill401
Ψ Lint check passed!
```

**Comment by fauxpark** [comment]
That's because you have a rules.mk at the root, which signals to the build system that this is a valid keyboard target, however there are things missing (namely the matrix pins) at this level since they are only in the "revision" folders.
You should either remove the root rules.mk, or set `DEFAULT_FOLDER = kinesis/kinbp/<rev>` to redirect it.

File: keyboards/kinesis/kinbp/blackpill401/info.json
Comment (fauxpark) [inline keyboards/kinesis/kinbp/blackpill401/info.json:1]: Please run `qmk format-json -i` on this file.

File: keyboards/kinesis/kinbp/blackpill411/info.json
Comment (fauxpark) [inline keyboards/kinesis/kinbp/blackpill411/info.json:1]: As above.

File: keyboards/kinesis/kinbp/info.json
Comment (fauxpark) [inline keyboards/kinesis/kinbp/info.json:1]: As above.

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by LXF-YZP — COMMENTED** [review]
`"usb": {
        "device_version": "0.0.1",
        "pid": "0x4B4E",
        "vid": "0xFEED"
    },`

vid better not. If it wasn't for 0xFEED,via wouldn't pass the review when it was submitted.

**Review by drashna — APPROVED** [review]

**Review by DmNosachev — COMMENTED** [review]
Hi all. Please tell me what I should do. It seems that I have made all the requested changes. As always, I lack experience with Github

**Pull request closed without merging**